cmd/earthly/main.go

```diff
@@ -3127,6 +3135,8 @@ func ifNilBoolDefault(ptr *bool, defaultValue bool) bool {
 }
 
 func (app *earthlyApp) actionListTargets(c *cli.Context) error {
+	app.commandName = "listTargets"
```

Comment:
noticed this was producing analytics event with key name `cli-` -> now it's `cli-listTargets`.